src/theming.jl

```diff
@@ -2,7 +2,7 @@
 const minimal_default = Attributes(
     font = "Dejavu Sans",
     backgroundcolor = RGBAf0(1,1,1,1),
-    color = :black,
+    color = Palette(ColorBrewer.palette("Dark2", 8), cycle = true),
```

Comment:
Is this necessary? Why not just `Palette(:Dark2)`?

Reply:
Because this is executed at precompilation time, so the `Palette(:Dark2)` method is not yet available. From the user side the two calls are equivalent.

Reply:
why does it matter that it is built at precompilation time? Sorry for being slow, but isn't that just a matter of changing the order of includes, or is something happen in precompilation that I'm not realising? Sorry if the question is stupid.

Reply:
It's a very legitimate question. I simply couldn't find a reshuffling of the "includes" that would allow me to call `Palette(:Dark2)` at this point of the code and didn't want to refactor the code of the package too much. Of course in say MakieThemes or no matter what other package, one would depend on an already precompiled AbstractPlotting and use all the utility functions.